cosmos/kima_network.json

```diff
@@ -0,0 +1,55 @@
+{
+    "chainId": "kima_network",
+    "chainName": "Kima Network",
+    "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kimanetwork/chain.png",
+    "rpc": "https://rpc.kima.network/",
+    "rest": "https://api.kima.network/",
+    "nodeProvider": {
+      "name": "Kima Network",
+      "email": "support@kima.network",
+      "website": "https://kima.network/"
+    },
+    "bip44": {
+      "coinType": 118
+    },
+    "bech32Config": {
+      "bech32PrefixAccAddr": "kima",
+      "bech32PrefixAccPub": "kimapub",
+      "bech32PrefixValAddr": "kimavaloper",
+      "bech32PrefixValPub": "kimavaloperpub",
+      "bech32PrefixConsAddr": "kimavalcons",
+      "bech32PrefixConsPub": "kimavalconspub"
+    },
+    "currencies": [
+      {
+        "coinDenom": "KIMA",
+        "coinMinimalDenom": "uKIMA",
+        "coinDecimals": 6,
+        "coinGeckoId": "kima"
```

Comment:
Since "kima" for the coinGeckoId doesn't seem available now on CoinGecko, please remove the lines from this file. 
https://www.coingecko.com/en/coins/kima
You can add coinGeckoIds later.

Reply:
I'll remove the `additionalEndPoints` but the token is out (blockchain is on mainnet) - it's just not tradeable yet, as we haven't announced our TGE. Hence the listing on CoinGecko saying "Preview Only".

Should I just remove the token fields from the mainnet JSON, and resubmit later when listed?

Reply:
Leave token informations but remove coinGeckoId fields only.

Reply:
Hi @editaahn - turns out you're right! Whoever listed our token on Coingecko listed just the Arbitrum one, and not the Cosmos one! It was an error, and they're working to fix. As soon as the right token is up, I'll resubmit. Thanks and sorry!